File: go/docmost/models/search_dto.go
package models

// SearchDto from OpenAPI.
type SearchDto struct {
	Query string `json:"query"`
	SpaceId *string `json:"spaceId,omitempty"`
	CreatorId *string `json:"creatorId,omitempty"`
	Limit *int `json:"limit,omitempty"`
	Offset *int `json:"offset,omitempty"`
}

File: c#/src/Docmost.Sdk/Models/SearchDto.cs
namespace Docmost.Sdk.Models;

public class SearchDto
{
    [System.Text.Json.Serialization.JsonPropertyName("query")]
    public string Query { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("spaceId")]
    public string? SpaceId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("creatorId")]
    public string? CreatorId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("limit")]
    public int? Limit { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("offset")]
    public int? Offset { get; set; }

}
